From 6935649a5a4c53ce4ec55e07210ed76b49d937d6 Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Wed, 9 Feb 2022 18:43:35 +0100
Subject: IWYU: missing include for usage of memset in
 SimplePartitionStatsDumper
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix build error with libstdc++:
    ../../base/allocator/partition_allocator/partition_stats.cc: In constructor ‘partition_alloc::SimplePartitionStatsDumper::SimplePartitionStatsDumper()’:
    ../../base/allocator/partition_allocator/partition_stats.cc:10:3: error: ‘memset’ was not declared in this scope
       10 |   memset(&stats_, 0, sizeof(stats_));
          |   ^~~~~~

Bug: 957519
Change-Id: I7f8297859f663c4ba58bae0429f33da9071bb0d7
---
 base/allocator/partition_allocator/partition_stats.cc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/base/allocator/partition_allocator/partition_stats.cc b/base/allocator/partition_allocator/partition_stats.cc
index a5835a879ff50..5ece588a0d049 100644
--- a/base/allocator/partition_allocator/partition_stats.cc
+++ b/base/allocator/partition_allocator/partition_stats.cc
@@ -4,6 +4,8 @@
 
 #include "base/allocator/partition_allocator/partition_stats.h"
 
+#include <cstring>
+
 namespace partition_alloc {
 
 SimplePartitionStatsDumper::SimplePartitionStatsDumper() {
-- 
2.32.0

